cli/azd/cmd/wire.go

```diff
@@ -0,0 +1,202 @@
+//go:build wireinject
```

Comment:
Maybe add a comment that you should run `go generate` after modifying this file to update the generated code?